spec/jobs/voice_otp_sender_job_spec.rb

```diff
@@ -14,14 +14,10 @@
 
       calls = FakeVoiceCall.calls
 
-      code = '1234'.scan(/\d/).join(', ')
-      message = t('jobs.voice_otp_sender_job.message', code: code)
-      url_message = URI.escape(message)
-
       expect(calls.size).to eq(1)
       call = calls.first
       expect(call.to).to eq('555-5555')
-      expect(call.url).to include(url_message)
+      expect(call.url).to include('code=1234')
```

Comment:
So now we're sending a URL to our own servers, and the response of that URL contains the message. The URL no longer contains the entire message anymore, so I figured I'd check for the most important part of the URL, which is the code parameter?